### DISCUSSION TITLE
Canais Palco

### DISCUSSION THREAD

**Discussion by ghost** [eritislami/evobot#1008] [General]
O Bot encerra quando um usuário move o Bot para um Canal Palco

**Comment by ItsLeon15** [comment]
Move the bot into a normal VC then Stage Channel and make the Bot the speaker and it should work. 

_You may need to edit the code so the bot accepts stage channels_

**Comment by wolf-yuan-6115** [comment]
> 
> 
> O Bot encerra quando um usuário move o Bot para um Canal Palco

Aguarde a reescrita de Discord.js v13, v13 aceite canal de estágio.
*Via Google translate*
